editor/src/app/header/header.component.ts

```diff
@@ -33,6 +33,31 @@ import { map } from 'rxjs/operators';
       justify-content: flex-end;
     }
 
+    header.loading:after {
```

Comment:
Looks good, but I think this solution with the after the element is a bit too complicated. We can simply add a `<div class="loading">` as the first child of the header, and show/hide it according to application loading state.